### ISSUE TITLE
`auth_settings_v2` on `azurerm_windows_web_app ` doesn't allow to set `0` value of `token_refresh_extension_time` login block field

### ISSUE THREAD

**Issue by michaelquintela** [hashicorp/terraform-provider-azurerm#21006]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

1.3.7

### AzureRM Provider Version

3.45.0

### Affected Resource(s)/Data Source(s)

azurerm_web_app

### Terraform Configuration Files

```hcl
resource "azurerm_windows_web_app" "web_app" {
  provider = azurerm.GSS_pay_as_you_go
  name = module.common.resource_name
  resource_group_name = module.common.resource_name
  location = data.azurerm_service_plan.app_service_plan.location
  service_plan_id = data.azurerm_service_plan.app_service_plan.id
  https_only = true
  tags = local.tags
  auth_settings_v2 {
    auth_enabled = true
    default_provider = "aad"
    runtime_version = "~1"
    unauthenticated_action = "RedirectToLoginPage"
    active_directory_v2{
      client_id = local.ad_app_id
      login_parameters = {}
      tenant_auth_endpoint = "https://login.microsoftonline.com/${module.common.ad_tenant_id}/v2.0"
      www_authentication_disabled  = false
    }
    login{
      token_store_enabled = true
      # issue: not allow to set 0.
      token_refresh_extension_time = 0
    }
  }
  site_config {}
}
```

### Debug Output/Panic Output

```shell
error: expected auth_settings_v2.0.login.0.token_refresh_extension_time to be atlist (1.0000), got (0.00000)
```

### Expected Behaviour

The provider should allow to set `0` value of `auth_settings_v2.0.login.0.token_refresh_extension_time`.

### Actual Behaviour

A validation error occurred when we set ` 0 ` value of `auth_settings_v2.0.login.0.token_refresh_extension_time`.

### Steps to Reproduce

Upgrade Azure on auth v2 from portal.
Now, run terraform plan or terraform apply command.
Observed, it’s through an error.

Note: Here, se can set `token_refresh_extension_time` value `0` from azure portal.

Please refer attached screenshot for more information:
![image](https://user-images.githubusercontent.com/128145733/225894380-f2b0c0a6-898a-414c-9a46-9fe554a9e2ce.png)

**Comment by michaelquintela** [comment]
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/windows_web_app#token_refresh_extension_time

**Comment by catriona-m** [comment]
Thanks for raising this issue - it looks like this has now been fixed in #21013 so this can be closed now. Thanks!
